packages/core/src/command/feature-flags/index.ts

```diff
@@ -0,0 +1,154 @@
+
+export async function getFeatureFlagValueCxValues<T extends keyof StringValueFeatureFlags>(
+  region: string,
+  ffTableName: string,
+  cxId: string,
+  featureFlagNames?: T[]
+): Promise<CxFeatureFlagStatus> {
+  const configContentValue = await getFeatureFlags(region, ffTableName);
+  const targetFeatureFlags =
+    featureFlagNames && featureFlagNames.length > 0
+      ? featureFlagNames
+      : stringValueFFsSchema.keyof().options;
+  let response: CxFeatureFlagStatus = {};
+  targetFeatureFlags.map(featureFlagName => {
+    const featureFlag = configContentValue[featureFlagName];
+    if (featureFlag) {
+      response = {
+        ...response,
+        [featureFlagName]: {
+          cxInFFValues: featureFlag.values.includes(cxId),
+          ffEnabled: featureFlag.enabled,
+        },
+      };
+    }
+  });
+  return response;
+}
+
+export async function getFeatureFlagValueStringArray<T extends keyof StringValueFeatureFlags>(
+  region: string,
+  ffTableName: string,
+  featureFlagName: T
+): Promise<StringValueFeatureFlags[T]> {
+  try {
+    const configContentValue = await getFeatureFlags(region, ffTableName);
+    return configContentValue[featureFlagName];
+  } catch (error) {
+    const msg = `Failed to get Feature Flag Value`;
+    const extra = { featureFlagName };
+    log(`${msg} - ${JSON.stringify(extra)} - ${errorToString(error)}`);
+    capture.error(msg, { extra: { ...extra, error } });
+    return { enabled: false, values: [] };
+  }
+}
+
+export async function getFeatureFlagValueBoolean<T extends keyof BooleanFeatureFlags>(
+  region: string,
+  ffTableName: string,
+  envName: string,
+  featureFlagName: T
+): Promise<BooleanFeatureFlags[T]> {
+  const configContentValue = await getFeatureFlags(region, ffTableName);
+  return configContentValue[featureFlagName];
+}
+
+/**
+ * Checks whether the specified feature flag is enabled.
+ *
+ * @returns true if enabled; false otherwise
+ */
+export async function isFeatureFlagEnabled(
+  featureFlagName: keyof BooleanFeatureFlags,
+  defaultValue = false
+): Promise<boolean> {
+  try {
+    const featureFlag = await getFeatureFlagValueBoolean(
+      Config.getAWSRegion(),
+      Config.getFeatureFlagsTableName(),
+      Config.getEnvType(),
+      featureFlagName
+    );
+    return featureFlag ? featureFlag.enabled : defaultValue;
+  } catch (error) {
+    const msg = `Failed to get Feature Flag Value`;
+    const extra = { featureFlagName };
+    log(`${msg} - ${JSON.stringify(extra)} - ${errorToString(error)}`);
+    capture.error(msg, { extra: { ...extra, error } });
+  }
+  return defaultValue;
+}
+
+/**
+ * Returns the list of customers that are enabled for the given feature flag.
+ *
+ * @returns Array of string values
+ */
+export async function getCxsWithFeatureFlagEnabled(
+  featureFlagName: keyof StringValueFeatureFlags
+): Promise<string[]> {
+  try {
+    const featureFlag = await getFeatureFlagValueStringArray(
+      Config.getAWSRegion(),
```

Comment:
any reason this is a param instead of just being used directly within getFeatureFlagValueStringArray() + similar funcs? They seem like the should always operate on the appropriate region given the execution context anyway, so we can simplify the call signature.

Reply:
I'm trying to minimize changes with these PRs, so mostly trying to "switch the underlying function's implementations"* to minimize code change and testing.

*as one can, since we didn't have interfaces/classes, so have to create new functions anyways.